freebites/product
# 102 post confirmation modal

# Description

Added a modal that pops up on the home page to confirm when the user makes a new post

## Issues
Resolve #102 

## Screenshots

<img src="https://github.com/user-attachments/assets/d851eb03-ecf4-44bd-a0a2-6b6b581fce9f" width=250>

## Test
- log in
- create a new post
- when you submit your post and come back to the home page the modal should pop up
- create another post
- make sure the modal pops up again

## Possible Downsides

The modal's entering animation sometimes jitters a little bit. The current system for making sure that the modal only appears once is kinda scuffed and requires the home index file to verify expo router segments which isn't amazing.
